ChildProject/pipelines/metrics.py

```diff
@@ -40,6 +41,8 @@ class Metrics(ABC):
     :type child_cols: str, optional
     :param period: time units to aggregate (optional); equivalent to ``pandas.Grouper`` freq argument.
     :type period: str, optional
+    :param segments: DataFrame or path to csv file of the segments to extract from, containing 'recording_filename', 'segment_onset' and 'segment_offset' columns. To use this option, the <by> option must be set to 'segments', can not be used along with options [recordings,period,from_time,to_time].
```

Comment:
" To use this option, the <by> option must be set to 'segments'. Also, this option cannot be combined with options [recordings,period,from_time,to_time]."